src/modules/map/components/CompareSlider.vue

```diff
@@ -10,44 +11,121 @@ const props = defineProps({
         default: window.innerWidth,
     },
 })
-
+const getMap = inject('getMap')
 const { clientWidth } = toRefs(props)
```

Comment:
client width doesn't seem to be updated correctly, as if I'm resizing the window the slider behave in a weird way.
Maybe you could directly get this info from the store, by reading `state.ui.width`?